jfrog_platform_self-hosted/CHANGELOG.md

```diff
@@ -1,4 +1,9 @@
-# CHANGELOG - jfrog_platform
+# CHANGELOG - jfrog_platform_self-hosted
+
+## [1.3.0] - Jun 26th, 2023
+* Modified installation steps for sending logs, metrics and violations data to Datadog
+* Supporting only OS/VM, docker and k8s forms of installation for JFrog Platform
+* Updated documentation
```

Comment:
Please update the changelog to use this format:

```suggestion
# CHANGELOG - JFrog Platform Self-Hosted

## 1.3.0 / 2023-08-11

***Fixed***:
* Modified installation steps for sending logs, metrics and violations data to Datadog
* Supporting only OS/VM, docker and k8s forms of installation for JFrog Platform
* Updated documentation

```

Reply:
This should resolve your validation issues!

Reply:
@anawishnoff I've updated the CHANGELOG and the validate repository is still not successful 😢

Reply:
Hey @MahithaB There was indeed an issue with `ddev` that got fixed and released. I've re-run the CI and the latest failure is vaild: its failing because the CODEOWNERS file will need to be updated to reflect the new integration folder name. Once thats updated the CI should pass again. Sorry for the inconvenience there!

Reply:
@nmuesch thank you. I've changed the CODEOWNERS file to the new integration folder name. I think there is a new error now. Can you please give me more information on that?

Reply:
@MahithaB The service_checks.json file is required even if its just an empty json object. Can you put that file back and add this to the manifest:

```
      "service_checks": {
        "metadata_path": "assets/service_checks.json"
      }
```

Reply:
@nmuesch Got it, added the service_checks back

Reply:
@nmuesch  Can you give info the next validation issue

Reply:
Looks like the folder rename requires the codecov file to need to be updated. From the error it looks like you can run `ddev validate ci --sync` and commit the changes to get past it

Reply:
@nmuesch can I get more info on the next error please

Reply:
This one looks in Ofeks territory so I may not be able to help with specifics. But I think it's because the configuration spec.yaml file is deleted and not in the new folder.

Reply:
@ofek can I get more information on the current error? Thanks!